### PR TITLE
Fix serialize

### PR DESCRIPTION
**Pull request by wwweert123**

### CODE REVIEW

File: src/main/java/seedu/financialplanner/investments/Stock.java [hunk 1]
```diff
@@ -17,10 +16,11 @@
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
-public class Stock implements Serializable {
+public class Stock {
     private static final Logger logger = Logger.getLogger("Financial Planner Logger");
     private static final String API_ENDPOINT = "https://www.alphavantage.co/query?function=SYMBOL_SEARCH&keywords=";
-    private static final String API_KEY = "LNKL0548PHY2F0QU";
+    ///private static final String API_KEY = "LNKL0548PHY2F0QU";
```
Comment (NeoMinWei) [inline src/main/java/seedu/financialplanner/investments/Stock.java:22]: remove commented code if necessary

File: src/main/java/seedu/financialplanner/storage/LoadData.java [hunk 2]
```diff
@@ -149,20 +151,20 @@ private static String getDescription(String[] split) {
...
+            stocksData = gson.fromJson(reader, new TypeToken<HashMap<String,Stock>>(){}.getType());
+        } catch (FileNotFoundException e) {
             ui.showMessage("Watchlist file not found... Creating");
-        } catch (ClassNotFoundException e) {
-            ui.showMessage("FIle appears to be corrupted...");
+        } catch (JsonSyntaxException e) {
+            ui.showMessage("Watchlist JSON is corrupted!");
+            ui.showMessage("Would you like to create new file? (Y/N)");
+            if (!createNewFile()) {
+                ui.showMessage("Exiting... Plese fix the file");
```
Comment (NeoMinWei) [inline src/main/java/seedu/financialplanner/storage/LoadData.java]: Plese

**Review by NeoMinWei — APPROVED** [review]
LGTM

File: src/main/java/seedu/financialplanner/storage/LoadData.java
Comment (ryan1604) [inline src/main/java/seedu/financialplanner/storage/LoadData.java:1]: Gj

**Review by ryan1604 — APPROVED** [review]
LGTM

**Pull request merged**